app/views/articles/show.html.erb

```diff
@@ -203,12 +202,31 @@
     <%= render "articles/full_comment_area" %>
   </div>
 
-# Line 199 ends the div holding the main section of the article
-# Somewhere below...I believe is a "classic dev post" followed by "another post you might like"
-# Depending on where we decide to put these videos would be either right here at the top or maybe below the suggested_articles
+<!-- Line 199 ends the div holding the main section of the article
+Somewhere below...I believe is a "classic dev post" followed by "another post you might like"
+Depending on where we decide to put these videos would be either right here at the top or maybe below the suggested_articles -->
 
   <% cache("article-bottom-content-#{@article.id}", expires_in: 18.hours) do %>
     <% @classic_article = Suggester::Articles::Classic.new(@article, not_ids: [@article.id]).get.first %>
+    <div class="container show-page-content-display">
```

Comment:
Plans to refactor lines 211-229 shortly

Reply:
we'll definitely be able to refactor this and break out into a partial that does all the iterating functionality we need to display 3 videos on the page.